src/game/AI/ScriptDevAI/scripts/world/scourge_invasion.cpp

```diff
@@ -469,7 +535,14 @@ struct NecropolisProxyAI : public ScriptedAI
             m_creature->ForcedDespawn();
     }
 
-    void UpdateAI(uint32 const diff) override {}
+    void UpdateAI(uint32 const diff) override
```

Comment:
I could make these CustomActions, but I don't know for what reason UpdateAI() was overridden to do nothing, so I kept it here instead. Might make sense to not run this every tick tho and instead use a timer.